.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
These two `.json` files don't belong to this PR. You can remove this commit, if you don't know how to do it I can show you.

Reply:
Yes, please let me know how to fix this.